src/OrleansServiceBus/Providers/Streams/EventHub/EventHubSequenceToken.cs

```diff
@@ -0,0 +1,33 @@
+﻿
+using System;
+using System.Globalization;
+using Orleans.Providers.Streams.Common;
+
+namespace Orleans.ServiceBus.Providers.Streams.EventHub
+{
+    /// <summary>
+    /// Event Hub messages consist of a batch of application layer events, so EventHub tokens contain three pieces of information.
+    /// Offset - this is a unique value per partition that is used to start reading from this message in the partition.
+    /// SequenceNumber - EventHub sequence numbers are unique ordered message IDs for messages within a partition.  
+    ///   The SequenceNumber is required for uniqueness and ordering of EventHub messages within a partition.
+    /// batch offset - Since each EventHub message may contain more than one application layer event, this value
```

Comment:
`batchOffset` or `this.EventIndex` ? They are the same, right? So maybe lets use same name?
But with already 3 different ones, having 2 names for the same one is too much. ;-)